### ISSUE TITLE
Filter backend schema inspector issue

### ISSUE THREAD

**Issue by matiucarr** [encode/django-rest-framework#5594]
## Checklist

- [x] I have verified that that issue exists against the `master` branch of Django REST framework.
- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] This is not a usage question. (Those should be directed to the [discussion group](https://groups.google.com/forum/#!forum/django-rest-framework) instead.)
- [x] This cannot be dealt with as a third party library. (We prefer new functionality to be [in the form of third party libraries](http://www.django-rest-framework.org/topics/third-party-resources/#about-third-party-packages) where possible.)
- [x] I have reduced the issue to the simplest possible case.
- [ ] I have included a failing test as a pull request. (If you are unable to do so we can still accept the issue.)

## Steps to reproduce

+ Using django_restframework 3.7.3 in conjunction with django-filter 1.1.0 and django-rest-swagger 2.1.2
+ configure a default filter backend as per the documentation in DRF and Djanjo-Filter __or__ add it using the filter_backends attribute of the view.
+ create a suitable filterset and add it to the view via the filter_class attribute.

## Expected behavior

Filter options should render under the appropriate method in swagger, etc.

## Actual behavior

+ filter fields do not render in the swagger view, or the bvrowseable API.

## a fix

+ override action in the view class -this is silly but provided a clue

+ change line 371 of restframework/schemas/inspectors.py to actually generate a lower case version of the method:

change

```python
return method.lower in ["get", "put", "patch", "delete"]

```

to

```python
return method.lower() in ["get", "put", "patch", "delete"]

```

**Comment by carltongibson** [comment]
Yep. That seems like a bug. (Wondering about test coverage there...) 

@matiucarr If you're up for it we'd happily see a PR (with a small failing test) to address this. 

Thanks for the report!

**Comment by D3X** [comment]
@carltongibson I've created a PR with a test and a fix here: #5613 
Let me know if that test makes sense in your opinion.

**Comment by rpkilby** [comment]
Thanks @D3X!
